1.8/development/index.md

```diff
@@ -11,48 +11,7 @@ The Mesosphere Datacenter Operating System (DC/OS) provides the optimal user exp
-
-# <a name="adminrouter"></a>Admin Router and web interface integration
-
-By default, a DC/OS service is deployed on a [private agent node](/docs/1.8/overview/concepts/#private-agent-node). To allow configuration control or monitoring of a service by a user, the Admin Router proxies calls on the master node to the service in a private node on the cluster. The HTTP service endpoint requires relative paths for artifacts and resources. The service endpoint can provide a web interface, a RESTful endpoint, or both. When creating a DC/OS CLI subcommand, it is common to have a RESTful endpoint to communicate with the scheduler service.
-
-The integration to the Admin Router is automatic when a framework scheduler registers a `webui_url` during the registration process with the Mesos master. There are a couple of limitations:
-
-*   The URL must NOT end with a forward slash (/). For example, this is good `internal.dcos.host.name:10000`, and this is bad `internal.dcos.host.name:10000/`.
-*   DC/OS supports 1 URL and port.
-
-When the `webui_url` is provided, the service is listed on the DC/OS web interface as a service with a link. That link is the Admin Router proxy URL name that is based on a naming convention of: `/service/<service_name>`. For example, `<dcos_host>/service/unicorn` is the proxy to the `webui_url`. If you provide a web interface, it will be integrated with the DC/OS web interface and users can click the link for quick access to your service.
-
-Service health check information is provided from the DC/OS service tab when:
-
-*   There are service health checks defined in the `marathon.json` file. For example:
-
-```json
-"healthChecks": [
-    {
-  "path": "/",
-  "portIndex": 1,
-  "protocol": "HTTP",
-  "gracePeriodSeconds": 5,
-  "intervalSeconds": 60,
-  "timeoutSeconds": 10,
-  "maxConsecutiveFailures": 3
-    }
-]
-```
-
-*   The `framework-name` property in the `marathon.json` file is valid. For example:
-    
-          "id": "{{kafka.framework-name}}"
-        
-
-*   The framework property in the `package.json` file is set to true. For example:
-    
-          "framework": true
-        
-
-You can provide public access to your service through the Admin Router or by deploying your own proxy or router to the public agent node. It is recommend to use the Admin Router for scheduler configuration and control, allowing integration with the DC/OS web interface. You can also provide a [CLI subcommand](/docs/1.8/development/cli-spec/) for command-line control of a RESTful service endpoint for the scheduler.
+All packaged services are required to meet a certain standard as defined by Mesosphere. For details on submitting a DC/OS service, see [Getting Started with Universe](https://github.com/mesosphere/universe/blob/version-3.x/docs/tutorial/GetStarted.md).
```

Comment:
@jsancio is it okay to rely on this url? I ask this because it includes a branch name in the url and if it changes in long term, it may not be good for SEO. Is it acceptable if we rely on sth. like https://github.com/mesosphere/universe/blob/HEAD/docs/tutorial/GetStarted.md ?

Reply:
I am okay using a different URL. How are we going to make it a different URL? Are you suggesting we move that markdown from `github.com/mesosphere/universe` to this repo?

Reply:
Probably. I don't know how Github defines `HEAD`.

Reply:
Cursory research on this hasn't been helpful -- should I change the URL even though we're not sure what `HEAD` is?

Reply:
@sascala if i am not mistaken, `HEAD` is defined as the latest commit. In the way that we are using, it points to the `latest` commit in the `default` branch. However, as it points to a commit instead of a bare repository, it may not be "aesthetically ideal". So lets just keep whatever you have currently and may be we can deal with thin in future if it starts being a pain.